### DISCUSSION TITLE
Any freelancers webdevs?

### DISCUSSION THREAD

**Discussion by rik1301** [sqlitebrowser/sqlitebrowser#3416] [Q&A]
Just wondering if there are any freelancer webdevs here?  I want to get my database online so can be queried by people but finding it impossible to find someone who will work with .db format so that all I have to do is drag and drop the file to my FTP.   The very few people who did show some interest wanted me to mess about converting it to csv or sql format first which I'm not prepared to do each time.  Is .db a format an impossible format to "connect" to a website?   All I want is to be able to query my database, have some conditional formatting row background shading and create some pages with filter views from the databases.

**Comment by justinclift** [comment]
>  Is .db a format an impossible format to "connect" to a website?

Nope.  SQLite support (the .db file format you're probably after) is built in to most hosting platforms in one way or another.

For the best possible audience though, you're probably best off asking on the SQLite Forum:

https://sqlite.org/forum/forum

There's liable to be a proverbial shed-load of web developers there. :smile:

**Comment by chrisjlocke** [comment]
It can't do everything you want, but have you seen DBHub.io?  It's not primarily aimed for your requirements, but hosting SQLite databases it does do very well.

> Is .db a format an impossible format to "connect" to a website?

With my pedantic hat on, there is no 'db' format.  .db is just a 'container'.  DB could be SQLite, MySQL, Postgres, etc.  Its hard having a 'one size fits all' interface for database files - a database can contain two tables or 5000.  Each table could contain two fields or 5000.  Each table can contain two records or millions.  Would you really scroll across 5000 columns?  Its easy to display an HTML table, but hard to filter and search one.  
It sounds simple - 'I want to display this file' but 'behind the scenes', it is a nightmare.
